app/assets/javascripts/discourse/app/components/topic-list/item/topic-cell.gjs

```diff
@@ -61,62 +61,72 @@ export default class TopicCell extends Component {
           @name="topic-list-before-status"
           @outletArgs={{hash topic=@topic}}
         />
-        {{~! no whitespace ~}}
-        <TopicStatus @topic={{@topic}} />
-        {{~! no whitespace ~}}
-        <TopicLink
-          {{on "focus" this.onTitleFocus}}
-          {{on "blur" this.onTitleBlur}}
-          @topic={{@topic}}
-          class="raw-link raw-topic-link"
-        />
-        {{~#if @topic.featured_link~}}
-          &nbsp;
-          {{~topicFeaturedLink @topic}}
-        {{~/if~}}
         <PluginOutlet
```

Comment:
for whitespace parity you'll need a `{{~! no whitespace ~}}` above this line

Reply:
Done